pkg/helpers/pki.go

```diff
@@ -88,7 +88,11 @@ func CreatePki(extraFQDNs []string, extraIPs []net.IP, clusterDomain string, caP
 	}
 
 	group.Go(func() (err error) {
-		apiServerCertificate, apiServerPrivateKey, err = createCertificate("apiserver", caCertificate, caPrivateKey, false, true, extraFQDNs, extraIPs, nil)
+		ip := net.ParseIP("127.0.0.1").To4()
```

Comment:
Please refer to my latest PR comment. If that is done, we won't need to do these kind of appends